cds_migrator_kit/migration_config.py

```diff
@@ -355,3 +355,4 @@ def _(x):  # needed to avoid start time failure with lazy strings
 base_path = os.path.dirname(os.path.realpath(__file__))
 logs_dir = os.path.join(base_path, "tmp/logs/")
 CDS_MIGRATOR_KIT_LOGS_PATH = logs_dir
+INVENIO_CDS_MIGRATOR_KIT_STREAM_CONFIG = "cds_migrator_kit/rdm/migration/streams.yaml"
```

Comment:
minor: I would remove invenio
```suggestion
CDS_MIGRATOR_KIT_STREAM_CONFIG = "cds_migrator_kit/rdm/migration/streams.yaml"
```